### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by TingTingin** [adobe-research/custom-diffusion#16]
I use this command with 100 reg images in ./data/1girl-reg and 56 instance images in ./data/Power

`accelerate launch src/diffuser_training.py --pretrained_model_name_or_path 'C:\Users\jhon\StableTuner\models\Anything-V3' --instance_data_dir ./data/Power --class_data_dir ./data/1girl-reg --output_dir ./logs/power --with_prior_preservation --prior_loss_weight 1.0 --instance_prompt "illustration of a <new1> 1girl" --class_prompt "1girl" --resolution 512 --revision fp16 --mixed_precision fp16 --gradient_checkpointing --gradient_accumulation_steps 1 --use_8bit_adam --train_batch_size 1 --learning_rate 5e-6 --lr_warmup_steps 0  --max_train_steps 250 --scale_lr  --modifier_token "<new1>"`

after training I sample like this
`python src/sample_diffuser.py --delta_ckpt logs/power/delta.bin --ckpt "C:\Users\jhon\StableTuner\models\Anything-V3" --prompt "illustration of <new1> 1girl"`

I also tried 

`python src/sample_diffuser.py --delta_ckpt logs/power/delta.bin --ckpt "C:\Users\jhon\StableTuner\models\Anything-V3" --prompt "<new1> 1girl"`

the generated images do not look anything like my instance character below are some of the the instance character images that I used for training 

![explorer_FRprcjSFmJ](https://user-images.githubusercontent.com/36141041/211209201-2876cf28-7a3d-423c-bc8a-aaa78c4c548b.jpg)

below are some of the regularization images that I used for training
![explorer_bMvWoV5Iz9](https://user-images.githubusercontent.com/36141041/211209250-ae94faeb-9303-4cc8-9154-733dc1707abf.jpg)

and this is the sampled image
![s](https://user-images.githubusercontent.com/36141041/211209272-39d158f7-b117-4822-b5ac-93c74bbfa53e.png)

As you can see the images look nothing like the instance character not sure what I am doing wrong

**Comment by nupurkmr9** [comment]
Hi, 
Thanks a lot for pointing out the error with mixed precision training. I will look into it more. 

Regarding enabling full fine-tune in the same code, it should be possible by adding another type in the `--freeze_model` flag, and enabling all params to have gradients in the `create_custom_diffusion` function. Also, in case of full fine-tuning calling `save_progress` during training and `load_model` during inference is not required. 
Let me know if you need more details. I will see if I can update the code to enable this as well. 

Thanks.
